html/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/StyleScope.kt

```diff
@@ -190,6 +194,7 @@ internal fun StylePropertyList.nativeEquals(properties: StylePropertyList): Bool
     return all { prop ->
         val otherProp = properties[index++]
         prop.name == otherProp.name &&
-            prop.value.toString() == otherProp.value.toString()
+            prop.value.toString() == otherProp.value.toString() &&
+                prop.important == otherProp.important
```

Comment:
```suggestion
            prop.important == otherProp.important && prop.value.toString() == otherProp.value.toString()
                
```

changing the order of the checks can help a bit - it's better to compare the booleans first.